internal/vendors/accessor_test.go

```diff
@@ -87,3 +88,141 @@ func Test_postgresVendorAccessor_GetSomeStuff(t *testing.T) {
 		g.Expect(res).To(gomega.BeNil())
 	})
 }
+
+func TestVendorAccessor_GetAll(t *testing.T) {
+	t.Parallel()
+
+	var (
+		accessor *postgresVendorAccessor
+		mock     sqlmock.Sqlmock
+	)
+
+	setup := func(t *testing.T) (*gomega.GomegaWithT, *sql.DB) {
+		g := gomega.NewWithT(t)
+		db, sqlMock, err := sqlmock.New(sqlmock.QueryMatcherOption(sqlmock.QueryMatcherEqual))
+		if err != nil {
+			log.Fatal("error initializing mock:", err)
+		}
+
+		accessor = newPostgresVendorAccessor(db)
+		mock = sqlMock
+
+		return g, db
+	}
+
+	sampleData := []string{
+		"id",
+		"name",
+		"bp_id",
+		"bp_name",
+		"rating",
+		"area_group_id",
+		"area_group_name",
+		"sap_code",
+		"modified_date",
+		"modified_by",
+		"dt",
+	}
+
+	query := `SELECT 
+			"id",
+			"name",
+			"bp_id",
+			"bp_name",
+			"rating",
+			"area_group_id",
+			"area_group_name",
+			"sap_code",
+			"modified_date",
+			"modified_by",
+			"dt" 
+			FROM vendor`
```

Comment:
I think you should also return description

Reply:
Yea, sorry I missed it. Thanks for the double check bro 👍